apdbPlots.py

```diff
@@ -699,7 +768,178 @@ def plotDiaSourcesOnSkyGrid(repo, sourceTable, title=None, color='C0'):
         ax.invert_xaxis()
         ax.get_xaxis().set_visible(False)
         ax.get_yaxis().set_visible(False)
-    fig.tight_layout()
+    fig.tight_layout(rect=[0, 0.03, 1, 0.95])
     plt.subplots_adjust(wspace=0)
     if title:
         fig.suptitle(title)
+
+
+def plotFlagHist(sourceTable, title=None, defaultBadFlags=True, colorList=None):
```

Comment:
Would it be possible to pass in a list of bad flag names, and programmatically color those red and the rest blue?

Reply:
Good point, that would be much more useful